### DISCUSSION TITLE
deepin-movie: 建议空格键暂停时，只出现中间播放按钮，无需出现标题栏和控制栏。鼠标移上再出现

### DISCUSSION THREAD

**Discussion by jay763190097** [linuxdeepin/developer-center#2945] [Features Request & Ideas | 特性请求 & 头脑风暴]

**Comment by sonald** [comment]
我觉得PM大概不会同意的，毕竟出现标题栏也是用户建议。。。

**Comment by jay763190097** [comment]
@sonald 暂停无需出现标题栏和进度条啊，而且进度条遮挡影响看字幕。鼠标移上原来的功能一样不影响啊。

**Comment by sonald** [comment]
我转PM看看讨论下

**Comment by jay763190097** [comment]
好的，谢谢

**Comment by jay763190097** [comment]
@sonald 讨论结果告知一下哦

**Comment by sonald** [comment]
@jay763190097 还没有结果，PM太忙了...

**Comment by jay763190097** [comment]
@zccrs ^_^，能否改成这样呢？改动我想应该很小

**Comment by wangjia12** [comment]
使用空格键暂停和播放时，只显示暂停按钮，不显示上下标题栏和控制栏；
鼠标移入影院时显示标题栏和控制栏；
空格键和鼠标行为单独处理；
@zccrs 15.8.3阶段做，这个任务不知道为什么不能设置里程碑。

**Comment by jay763190097** [comment]
非常棒

**Comment by aiamuzz** [comment]
@jay763190097 ... true ... the control bar does obscure the subtitle at the bottom ...

@wangjia12 ...

As deepin-movie full screen mode disables the deepin desktop corner actions and the dock ... wouldn't it be great to create deepin-movie's own corner settings ?

1. **bottom-right - could open/close playlist with the playlist disappearing all be itself after say 5 secs**

- _currently one has to 'click' on the playlist button and a second 'click' outside the playlist area for it to hide ... **Where as a bottom-right corner setting for playlist(the same way control center functions on the desktop)  will eliminate both the clicks all together ... making the use case more simple and efficient**._


2. **the bottom edge of the screen which the dock uses(but is not functional/deactivated when deepin-movie is in full screen mode) could activate the control bar and the title bar**

- _for viewing the name of the movie playing ... i know it sounds ridiculous ... But when i am watching movies i sometimes forget the name(which also has the year the movie was made) ... :) ... without an option to view the same on a full screen movie player ... i have to hit Alt+tab to go to file manager just to know the name of the movie playing._

- _Also the accidental nudge to the mouse on a cluttered desk or the bed brings about the control bar ... spoiling the whole full screen viewing experience ... **where as a bottom edge activation of the title and control bar will provide a disturbance free full screen viewing experience immune to mouse being moved accidentally.**_

and like @wangjia12 ... **was suggesting ... only the play / pause button is displayed.**

efficiency and ease of use ... eliminating extra steps in the process flow should be the way forward in designing applications ... don't you think ?

**Comment by wangjia12** [comment]
> 1. the bottom edge of the screen which the dock uses(but is not functional/deactivated when deepin-movie is in full screen mode) could activate the control bar and the title bar

Thank you for your suggestion and we will seriously consider it.

**Comment by aiamuzz** [comment]
> > 1. the bottom edge of the screen which the dock uses(but is not functional/deactivated when deepin-movie is in full screen mode) could activate the control bar and the title bar
> 
> Thank you for your suggestion and we will seriously consider it.

thanks for considering ... i hope this will enable a full screen experience without an accidental distraction (especially when watching a movie on the bed, the mouse just tends to move distracting the full screen) ... with the bottom edge as the trigger for bringing up the movie player controls is a good step forward.

I hope **in your next cycle** you'd consider the movie playlist to replicate the **control center behaviour / protocol ... which is the bottom right corner and disappear after 5-10 secs of the cursor moving out of the playlist column**

**Comment by jay763190097** [comment]
@zccrs 这个需求怎么还没改呢？每次看视频都遇到这个问题，新版在deepin V20和UOS仍然有这个问题。moonplayer mpv bomi等播放器都是空格键暂停不显示进度栏的

**Comment by meiyixiang** [comment]
这个问题我会反馈给产品，但会综合考虑交互和用户的意见。
